File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        Arrays.sort(nums);
        int n = nums.length;
        for (int i = 0; i < n; i++)
            System.out.print(nums[i] + " ");
        return nums[n - k];
    }
}

File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.py
from queue import PriorityQueue
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        pq = PriorityQueue()
        for i in nums:
            pq.put((-i, i))

        for i in range(k - 1):
            pq.get()
            
        return pq.get()[1]